### PR TITLE
The toolbar

### PR DESCRIPTION
**Pull request by bronnie313**
Testing the preparedness of the repository
added linter
added two sections both the header and the body section
Added my CSS to the project

### CODE REVIEW

File: toolbar.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <!-- header start -->
+    <div class="section">
+        <div class="container Logo">My Logo</div>
+        
+         <div><i id="menu" class="material-icons">menu</i></div>
+    </div>
+    <!-- header end -->
+
+    <!-- me section start -->
+    <div class="section_1">
```
Comment (bobb-Rob) [inline toolbar.html:21]: - [ ] The container of the headline section should be a `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

File: toolbar.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <div>
+            <h2 class="connect">LET'S CONNET</h2>
+            <div class="flex-icons">
+                <div><img src="twitter.jpg" alt="twitter"></div>
+                <div><img src="Linkedin icon.svg" alt="linked"></div>
+                <div><img src=m.jpg></div>  
+                <div><img src="git.jpg" alt="git"></div>
+                <div><img src="peace.jpg" alt="peace"></div>
+                
+            </div>
```
Comment (bobb-Rob) [inline toolbar.html:38]: - [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly fix this.

File: toolbar.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    <div class="section">
+        <div class="container Logo">My Logo</div>
+        
+         <div><i id="menu" class="material-icons">menu</i></div>
+    </div>
+    <!-- header end -->
+
+    <!-- me section start -->
+    <div class="section_1">
+        <div class="glad"><h1 class="myname">I’m Bronnie </br>Glad to see</br> you!</h1></div>
```
Comment (bobb-Rob) [inline toolbar.html:22]: - [ ] Please use the correct font-family as provided in the Figma template. The font should be `Poppins`.

File: toolbar.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        <p class="about">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```
Comment (bobb-Rob) [inline toolbar.html:23]: - [ ] Please use the correct font-family as provided in the Figma template. The font should be `Poppins`.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @bronnie313 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._
#### Please ensure you update your README file using the [new microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md).
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: toolbar.html (hunk 4 above)
Comment (bronnie313) [reply toolbar.html:23]: font-family Poppins continuously brought errors while using stylint hence using font-family: "Poppins", sans-serif

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @bronnie313  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:
- Linters are passing
- Project uses Figma template correctly

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**